ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -66,41 +64,38 @@ class ListDemoController: DemoController {
         }
 
         /// Custom Leading View with collapsible children items
-        listSection = MSFListSectionState()
-        listSection.title = "Avatar Section"
-        listSection.cells = []
+        let collapsibleSection = list.state.createSection()
+        collapsibleSection.title = "AvatarSection"
         for index in 0...1 {
-            listCell = MSFListCellState()
+            listCell = collapsibleSection.createCell()
             avatar = createAvatarView(size: .medium,
                                       name: samplePersonas[index].name,
                                       image: samplePersonas[index].image,
                                       style: .default)
             listCell.title = avatar.state.primaryText ?? ""
             listCell.leadingUIView = avatar.view
-            listSection.cells.append(listCell)
-            listSection.hasDividers = true
+        collapsibleSection.hasDividers = true
         }
-        listSection.cells[0].children = children
-        listSection.cells[0].isExpanded = true
-        listSection.cells[1].onTapAction = {
+        collapsibleSection.getCellState(at: 0).children = children
+        collapsibleSection.getCellState(at: 0).isExpanded = true
+        collapsibleSection.getCellState(at: 1).onTapAction = {
             self.showAlertForAvatarTapped(name: samplePersonas[1].name)
         }
-        listData.append(listSection)
 
         /// TableViewCell Sample Data Sections
         for sectionIndex in 0...sections.count - 1 {
```

Comment:
nit: opportunity for a more readable code:

`for sectionIndex in 0..<sections.count {`

(same on line 94 and other places you can find it)